pkg/operator/sync_controllermanager_v311_00.go

```diff
@@ -219,11 +219,13 @@ func manageServiceCatalogControllerManagerTrustedCAConfigMap_v311_00_to_latest(k
 	if apierrors.IsNotFound(err) {
 		newConfig, err := client.ConfigMaps(targetNamespaceName).Create(trustedCAConfigMap)
 		if err != nil {
-			recorder.Eventf("ConfigMapCreateFailed", "Failed to create %s%s/%s-n %s: %v", "configmap", "", "trusted-ca-bundle", targetNamespaceName, err)
+			recorder.Eventf("ConfigMapCreateFailed", "Failed to create %s/%s-n %s: %v", "configmap", "trusted-ca-bundle", targetNamespaceName, err)
 			return nil, true, err
 		}
-		recorder.Eventf("ConfigMapCreated", "Created %s%s/%s-n %s because it was missing", "configmap", "", "trusted-ca-bundle", targetNamespaceName)
+		recorder.Eventf("ConfigMapCreated", "Created %s/%s-n %s because it was missing", "configmap", "trusted-ca-bundle", targetNamespaceName)
 		return newConfig, true, nil
+	} else if err != nil {
+		return trustedCAConfigMap, false, err
```

Comment:
Should we return `nil, false, err` here so we don't give the impression that `trustedCAConfigMap` is present in the cluster?
```suggestion
		return nil, false, err
```